### DISCUSSION TITLE
Unable to Generate new key

### DISCUSSION THREAD

**Discussion by dohspc** [dgreif/ring#885] [Q&A] [answered]
Unable to generate a new key.  This error comes up no matter how I try and generate. It was working previously. I can log into my ring account fine. 

×
Ring Login Failed
Cannot use 'in' operator to search for 'error' in <html>
 <head><title>406 Not Acceptable</title></head>
 <body bgcolor="white">
 <center><h1>406 Not Acceptable</h1></center>
 <hr><center>nginx</center>
 </body>
 </html>

**Comment by dgreif** [comment]
Interesting, haven't seen this before.  Do you have 2fa turned on?  If so, what do you use for your second authentication mechanism (text, authenticator app, etc)?

**Comment by dohspc** [comment]
Yes via sms. It was working previously so not sure if ring changed something. You can’t disable 2FA anymore. Ring requires it. I can log into their website fine.

**Reply by dgreif** [reply]
Weird, just tried it with SMS and it's working fine for me.  It's possible your account is getting rate limited for some reason.  Also make sure you are on the latest version of the plugin

**Comment by dohspc** [comment]
I tried changing over to the authenticator app same error. Reset password same error.  Using the latest version of the plugin. Uninstalled/Reinstalled same error.  I can log into the ring.com website and app fine.  It was working fine until this weekend.

**Comment by dohspc** [comment, marked as answer]
I think their server was temporarily blocking my server authenticating because I just updated it with a static ip from my provider. I tried just now and was able to authenticate and got the cam working again. Closing the bug. Thanks for your work on this project.
